docs/docs/addons-pure-render-mixin.md

```diff
@@ -1,17 +1,29 @@
 ---
 id: pure-render-mixin
 title: PureRenderMixin
-permalink: docs-old/pure-render-mixin.html
-prev: update.html
-next: perf.html
+permalink: docs/pure-render-mixin.html
+layout: docs
+category: Add-Ons
 ---
 
-If your React component's render function is "pure" (in other words, it renders the same result given the same props and state), you can use this mixin for a performance boost in some cases.
+> Note:
+> `PureRenderMixin` is a legacy add-on. Use [`React.PureComponent`](/react/docs/react-api.html#react.purecomponent) instead.
+
+**Importing**
+
+```javascript
+import PureRenderMixin from 'react-addons-pure-render-mixin' // ES6
+var PureRenderMixin = require('react-addons-pure-render-mixin') // ES5 with npm
+var PureRenderMixin = React.addons.PureRenderMixin; // ES5 with react-with-addons.js
+```
+
+## Overview
+
+If your React component's render function renders the same result given the same props and state, you can use this mixin for a performance boost in some cases.
 
 Example:
 
 ```js
-var PureRenderMixin = require('react-addons-pure-render-mixin');
 React.createClass({
   mixins: [PureRenderMixin],
 
```

Comment:
Let's completely remove ES6 example below because it is irrelevant now that we have `PureComponent`.